src/ws/ws-server.js

```diff
@@ -210,8 +210,8 @@ function processNewConnection(ws) {
             addToLobby(ws, participantId, sessionId);
             phoenix.send(stateService.sessionJoin(sessionId, participantId));
         })
-        .catch((error) => {
-            error('[ws-server]', 'New connection rejected', error);
+        .catch((err) => {
```

Comment:
I'm fine with keeping it like that.
I just like namespacing better, because it makes things a bit more readable as you understand better where identifiers come from. And of course it resolves naming conflicts.